src/bot.ts

```diff
@@ -147,23 +172,20 @@ dfusionService.watchOrderPlacement({
 
     // Format the amounts
     // TODO: Allow to use BN, string or BigNumber or all three in the format. Review in dex-js
-    const sellAmountFmt = formatAmount(new BN(priceDenominator.toString()), sellToken.decimals)
-    const buyAmountFmt = formatAmount(new BN(priceNumerator.toString()), buyToken.decimals)
     const fillSellAmountFmt = formatAmountFull(
       new BN(priceNumerator.multipliedBy(FACTOR_TO_FILL_ORDER).toString()),
       buyToken.decimals
     )
     const buyAmountFullFmt = formatAmountFull(new BN(priceDenominator.toString()), sellToken.decimals)
 
-    // Compose message using markdown
-    // TODO: Provide the link to the front end: https://github.com/gnosis/dex-telegram/issues/3
     // TODO: Should we publish even if the user doesn't have balance. Should we include the balance of the user? he can change it...
+    //  https://github.com/gnosis/dex-telegram/issues/45
     const message = `Sell *${sellAmountFmt}* \`${sellTokenLabel}\` for *${buyAmountFmt}* \`${buyTokenLabel}\`
 
   - *Price*:  1 \`${sellTokenLabel}\` = ${price} \`${buyTokenLabel}\`
 ${datesDescription}
 
-Fill the order here: ${WEB_BASE_URL}/trade/${buyTokenLabel}-${sellTokenLabel}?sell=${fillSellAmountFmt}&buy=${buyAmountFullFmt}` // TODO:
+Fill the order here: ${WEB_BASE_URL}/trade/${sellTokenParam}-${buyTokenParam}?sell=${fillSellAmountFmt}&buy=${buyAmountFullFmt}`
```

Comment:
Can you create a task on `dex-react` to accept address as well as symbol?

Reply:
https://github.com/gnosis/dex-react/issues/279